examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -468,7 +468,7 @@
    "source": [
     "%%time\n",
     "item_features_names = ['f_' + str(col) for col in [47, 68]]\n",
-    "cat_features = [['item_id', 'purchase_id']] + item_features_names >> nvt.ops.Categorify(start_index=1)\n",
```

Comment:
May be we can add a note for the user something like that:

```
Categorify op maps nulls to `1`, OOVs to `2` and the frequent categories are encoded starting from `3`. Please note that we reserve `0` for padding value.
```

Reply:
can I propose merging this fix to get things working then you can update the copy?